### PR TITLE
Upgrade to Ember CLI 0.2.3

### PR DESCRIPTION
**Pull request by joshforisha**

### CODE REVIEW

**Comment by joshforisha** [comment]
I updated the addon install command in the README and the application.hbs.

File: .editorconfig [hunk 1]
```diff
@@ -18,6 +18,7 @@ indent_style = space
 indent_size = 2
 
 [*.hbs]
+insert_final_newline = false
 indent_style = space
 indent_size = 2
```
Comment (notmessenger) [inline .editorconfig]: https://github.com/softlayer/sl-ember-test-helpers/pull/72/files changed "indent_size" property throughout this file in that repo but the same changes were not made here.  We should be consistent across the repositories in this regard.

File: tests/unit/components/sl-translate-test.js [hunk 2]
```diff
@@ -1,43 +1,39 @@
...
- * that core Ember functionality is being tested this test is ensuring that the implied contract about how this non-UI
+ * Ensures that the template is wrapping the content in a span tag and not in
+ * any block-level tags. While it appears that core Ember functionality is being
+ * tested this test is ensuring that the implied contract about how this non-UI
  * component is rendered into the DOM is adhered to.
  */
 test( 'Renders as a span tag with no classes', function( assert ) {
-    var component  = this.subject({ translateService: translateService }),
-        $component = this.render();
+    var component = this.subject({ translateService });
```
Comment (notmessenger) [inline tests/unit/components/sl-translate-test.js]: Do not need assignment to var

File: tests/unit/components/sl-translate-test.js [hunk 3]
```diff
@@ -1,43 +1,39 @@
...
 /**
  * That it renders and functions as expected
  */
 test( 'DOM and content of rendered translation', function( assert ) {
-    var component = this.subject({
-            key              : 'the_key',
-            translateService : translateService
-        }),
-        $component = this.render();
+    var component = this.subject({ key: 'the_key', translateService });
```
Comment (notmessenger) [inline tests/unit/components/sl-translate-test.js]: Do not need assignment to var

File: tests/unit/components/sl-translate-test.js [hunk 4]
```diff
@@ -46,16 +42,15 @@ test( 'DOM and content of rendered translation', function( assert ) {
  */
 test( 'Can be used alongside other properties or attribute bindings', function( assert ) {
     var component  = this.subject({
```
Comment (notmessenger) [inline tests/unit/components/sl-translate-test.js]: Do not need assignment to var

File: tests/unit/components/sl-translate-test.js [hunk 5]
```diff
@@ -98,15 +93,15 @@ test( 'setTranslatedString() sets translatedString property with value from tran
 
 test( 'translateString() calls translateKey() on the translation service', function( assert ) {
     var component = this.subject({
```
Comment (notmessenger) [inline tests/unit/components/sl-translate-test.js]: Do not need assignment to var

**Comment by notmessenger** [comment]
https://github.com/softlayer/sl-ember-translate#binding needs to be changed to reflect the "Binding" changes

File: package.json [hunk 6]
```diff
@@ -28,25 +28,24 @@
...
-    "ember-cli-qunit": "0.3.7",
+    "ember-cli-qunit": "0.3.10",
     "ember-cli-uglify": "1.0.1",
-    "ember-data": "1.0.0-beta.14.1",
     "ember-export-application-global": "^1.0.2",
-    "express": "^4.8.5",
-    "glob": "^4.5.1",
-    "sl-ember-test-helpers": "1.3.0"
+    "ember-disable-prototype-extensions": "^1.0.0",
+    "ember-try": "0.0.4",
```
Comment (notmessenger) [inline package.json]: Remove this entry

**Pull request merged**